### DISCUSSION TITLE
get errors data in debug node

### DISCUSSION THREAD

**Discussion by lifef1** [alexandrainst/node-red-contrib-postgresql#35] [Q&A] [answered]
can i get errors data (incorrect query, timeout from database, etc) in debug node? or only from catch node? it is not comfortable. thanks for the answer.

**Comment by Alkarex** [comment, marked as answer]
Hello,
This node does provide error information that can be seen in debug node.

Example:

![image](https://user-images.githubusercontent.com/1008324/224006779-e0d57ba0-6819-464d-a9df-b4e18d04a487.png)

---

![image](https://user-images.githubusercontent.com/1008324/224006654-6e96ef19-4aa9-45d7-ad00-5f19a054e88d.png)

---

![image](https://user-images.githubusercontent.com/1008324/224006470-55a741a5-d1e5-4ff6-bd13-1fbbf6112325.png)
